### ISSUE TITLE
Do you use a different loss function?

### ISSUE THREAD

**Issue by jaelgu** [stdio2016/pfann#9]
Is the loss function different from the original paper? Also the method to calculate validate score?

**Comment by stdio2016** [comment]
I use NT-Xent loss function, the same as the original paper.
The validation score during training should be 1-second query accuracy, however it turned out to be way too high because I reused training pipeline for validation query generator. So I did not include the validation score from training process in my thesis.
Instead I wrote another query generator `genquery.py` that more accurately simulates the query process, and only computed accuracy during testing.
